### PR TITLE
Package apronext.1.0

### PR DESCRIPTION
**Pull request by ghilesZ**
### `apronext.1.0`
Apron extension
An extension for the OCaml interface of the Apron library

---
* Homepage: https://github.com/ghilesZ/apronext
* Source repo: git+https://github.com/ghilesZ/apronext
* Bug tracker: https://github.com/ghilesZ/apronext/issues

---
:camel: Pull-request generated by opam-publish v2.0.2

### CODE REVIEW

File: packages/apronext/apronext.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+dev-repo: "git+https://github.com/ghilesZ/apronext"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+run-test: [
+  ["dune" "runtest" "-p" name "-j" jobs]
+]
+depends: [
+  "dune" {build}
```
Comment (mseri) [inline packages/apronext/apronext.1.0/opam:15]:
```suggestion
  "dune" {>= "2.1"}
```
Your dune-project file requires dune 2.1

Comment (mseri) [reply packages/apronext/apronext.1.0/opam]: Can you make this fix upstream?

Comment (ghilesZ) [reply packages/apronext/apronext.1.0/opam]: Ok thanks

**Comment by mseri** [comment]
Thanks

**Pull request merged**